public/js/embedListeners.js

```diff
@@ -96,6 +96,21 @@ function startListeners() {
             }
 
         }
+        if (simulationArea.lastSelected && simulationArea.lastSelected.keyDown2) {
+            if (e.key.toString().length == 1) {
+                simulationArea.lastSelected.keyDown2(e.key.toString());
+                return;
+            }
+
+        }
+
+        // if (simulationArea.lastSelected && simulationArea.lastSelected.keyDown3) {
```

Comment:
Is it possible to remove this comment if not required?

Reply:
This was actually discussed last time it isn't required right now but might be required in future.